pkgs/applications/misc/cura/default.nix

```diff
@@ -61,6 +61,6 @@ mkDerivation rec {
     homepage = "https://github.com/Ultimaker/Cura";
     license = licenses.lgpl3Plus;
     platforms = platforms.linux;
-    maintainers = with maintainers; [ abbradar gebner ];
```

Comment:
Did you intend to remove more maintainers than just `gebner`? There seem to be many unnecessary maintainer removals.

Reply:
No more.